var/spack/repos/builtin/packages/foam-extend/package.py

```diff
@@ -87,6 +87,7 @@ class FoamExtend(Package):
     # mgridgen is statically linked
     depends_on("parmgridgen", when="+parmgridgen", type="build")
     depends_on("paraview@:5.0.1", when="+paraview")
+    depends_on("mesquite")
```

Comment:
Do you know if this dependency applies to other listed versions or for a specific version on?